srcpkgs/mvwm-git/patches/0001-Fix-cross-build.patch

```diff
@@ -0,0 +1,177 @@
+From 53b028752d6fdd96471be8cd4c3d1c8d490df2a5 Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Fri, 25 Dec 2020 19:45:56 -0600
+Subject: [PATCH] Fix cross-build
```

Comment:
Have you tried to upstream this? Would be nice to not have to rebase and add new changes when there are updates.

Reply:
Not yet. Add this to my TODO list :)

Reply:
Eh, upstream seems inactive since 2015... I'd consider dropping.

Reply:
Still, the patch likely won't become stale, so it wouldn't be so bad to add it.